tools/topology/topology1/sof/pipe-rtnr-google-rtc-audio-processing-capture.m4

```diff
@@ -0,0 +1,112 @@
+# Acoustic echo cancelling Pipeline and PCM
```

Comment:
Could you rename the file 
`pipe-google-rtc-audio-processing-rtnr-capture.m4` to reflect the processing pipeline ?
AEC->RTNR->Capture

Reply:
ack